File: KingdomCodeTool/wwwroot/Download/202410291354/Angular/shared/DanhMucNgonNgu.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucNgonNgu } from 'src/app/shared/DanhMucNgonNgu.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucNgonNguService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucNgonNgu";
    }
}

File: KingdomCodeTool/wwwroot/Download/202410291354/Service/Implement/DanhMucNgonNguService.cs
namespace Service.Implement
{
    public class DanhMucNgonNguService : BaseService<DanhMucNgonNgu, IDanhMucNgonNguRepository>
    , IDanhMucNgonNguService
    {
    private readonly IDanhMucNgonNguRepository _DanhMucNgonNguRepository;
    public DanhMucNgonNguService(IDanhMucNgonNguRepository DanhMucNgonNguRepository) : base(DanhMucNgonNguRepository)
    {
    _DanhMucNgonNguRepository = DanhMucNgonNguRepository;
    }
    }
    }

